docs/api.md

```diff
@@ -2211,6 +2198,76 @@ Verify a session using a recovery code.
   <!--end-response-body-post-sessionverifyrecoverycode-remaining-->
 
 
+### Recovery keys
+
+#### POST /recoveryKeys
+
+:lock: HAWK-authenticated with session token
+<!--begin-route-post-recoverykeys-->
+Creates a new recovery key for a user. These are one time use
+keys.
+<!--end-route-post-recoverykeys-->
+
+##### Request body
+
+* `recoveryKeyId`: *string, max(32), required*
+
+  <!--begin-request-body-post-recoverykeys-recoveryKeyId-->
+  Recovery Key Id is a hash of the recovery code.
```

Comment:
Saying "code" here is going to lead to all kinds of confusion with "recovery codes" :-)